catppuccin-steam.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash 
+
+# Install dependencies
+if [ -f /usr/bin/pacman ] 
+then
+  sudo pacman -S curl wget unzip git
+elif [ -f /usr/bin/emerge ]
+then
+  sudo emerge -av curl wget unzip git
+elif [ -f /usr/bin/apt ]
+then
+   sudo apt install curl wget unzip git
+elif [ -d /nix ]
+then
+  sudo nix-shell -p curl wget unzip git
+elif [ -f /usr/bin/dnf ]
+then
+  sudo dnf install curl wget unzip git
+fi
+
+# Make temp directory
+cd $(mktemp -d)
+
+# Get the latest release of SFP and unzip it
+mkdir SFP 
+cd SFP 
+curl -s https://api.github.com/repos/PhantomGamers/SFP/releases/latest \
```

Comment:
We shouldn't be downloading just the latest version, instead set a `SPF_VERSION` variable and download the specific version to avoid breaking changes. This also eliminates the need for both curl and wget

Reply:
honestly, that would probably break it even more, especially if there's a new release